test/lib/test_runner.go

```diff
@@ -213,6 +214,27 @@ func TearDown(client *Client) {
 	}
 }
 
+func formatEvent(e *corev1.Event) string {
+	return strings.Join([]string{`Event{`,
+		`ObjectMeta:` + strings.Replace(strings.Replace(e.ObjectMeta.String(), "ObjectMeta", "v1.ObjectMeta", 1), `&`, ``, 1),
+		`InvolvedObject:` + strings.Replace(strings.Replace(e.InvolvedObject.String(), "ObjectReference", "ObjectReference", 1), `&`, ``, 1),
+		`Reason:` + e.Reason,
+		`Message:` + e.Message,
+		`Source:` + strings.Replace(strings.Replace(e.Source.String(), "EventSource", "EventSource", 1), `&`, ``, 1),
+		`FirstTimestamp:` + e.FirstTimestamp.String(),
+		`LastTimestamp:` + e.LastTimestamp.String(),
+		`Count:` + string(e.Count),
```

Comment:
This doesn't seem to work. e.Count is an int32 and string() doesn't seem to do the right thing here.

Reply:
Yeah i noticed it and i fixed it but i forgot to push the update

Reply:
now i pushed the update